heimdall-frontend/src/containers/Middlewares.js

```diff
@@ -0,0 +1,175 @@
+import React, {Component} from 'react'
+import {connect} from 'react-redux'
+import {Upload, message, Icon, Input, Row, Card, Table, Pagination, Tag} from 'antd'
+import {save, initLoading, getMiddlewares} from '../actions/middlewares'
+
+const Dragger = Upload.Dragger
+const Column = Table.Column
+
+class Middlewares extends Component {
+
+    state = {
```

Comment:
@dijalmasilva  you add state's that aren't used. plz, remove they

Reply:
Fixed.

Reply:
Fixed